llama.cpp

```diff
@@ -104,6 +104,10 @@
 #define LLAMA_MAX_NODES   8192
 #define LLAMA_MAX_EXPERTS 8
 
+#if (defined(GGML_USE_CUBLAS) || defined(GGML_USE_SYCL))
+#define GGML_USE_CUBLAS_SYCL
+#endif
+
```

Comment:
`GGML_USE_CUBLAS_SYCL` appears to be unused

Reply:
yes, rm it.